test/unit/elasticsearch_index_test.rb

```diff
@@ -126,7 +126,7 @@ def test_commit
   end
 
   def test_can_fetch_documents_by_format
-    search_pattern = "http://example.com:9200/mainstream_test/_search?scroll=60m&search_type=scan&size=500"
+    search_pattern = "http://example.com:9200/mainstream_test/_search?scroll=1m&search_type=scan&size=500"
```

Comment:
It's just the domain that's configured in the unit test, so it seems OK to me to include it when the request is stubbed. Is there some problem that you think it might cause in the future?